Sources/PackageCollections/Model/Package.swift

```diff
@@ -17,11 +17,11 @@ import SourceControl
 extension PackageCollectionsModel {
     /// Package metadata
     public struct Package: Codable, Equatable {
-        /// Package reference
-        public let reference: PackageReference
+        /// Package identity
+        public let identity: PackageIdentity
 
-        /// Package's repository address
-        public let repository: RepositorySpecifier
+        /// Package location
+        public let location: String
 
```

Comment:
We need to keep `reference` and `repository` until folks switch over to `identity` and `location`.

Reply:
@yim-lee  on the Package model?

Reply:
All of the models. rdar://83113368, rdar://83114538 need to be done before we can remove `reference` and `repository`.

Reply:
@yim-lee does this help: https://github.com/apple/swift-package-manager/pull/3743/commits/dcb79b3a5608886d372cdea2b05b8b232258f569

Reply:
@tomerd yes